### PR TITLE
Add OSD Node.js compatibility and instructions for using alternative version of Node.js runtime binary

### PR DESCRIPTION
**Pull request by AMoo-Miki**
### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _install-and-configure/install-dashboards/index.md [hunk 1]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
...
+   ```
+
+   On Windows, if Node.js is installed to `C:\Program Files\nodejs` and the runtime binary is `C:\Program Files\nodejs\node.exe`:
+   ```powershell
+   set "NODE_HOME=C:\Program Files\nodejs"
+   # or using PowerShell:
+   $Env:NODE_HOME = 'C:\Program Files\nodejs'
+   ```
+
+   Please consult your operating system's documentation to make a persistent change to the environment variables.
```
Comment (kolchfa-aws) [inline _install-and-configure/install-dashboards/index.md:63]:
```suggestion
   Consult your operating system's documentation to make a persistent change to the environment variables.
```

File: _install-and-configure/install-dashboards/index.md [hunk 2]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
 
 Other Chromium-based browsers might work, as well. Internet Explorer and Microsoft Edge Legacy are **not** supported.
 
+## Node.js compatibility
+
+OpenSearch Dashboards requires the Node.js runtime binary to run and one is included in the distribution packages available from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}.
```
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md:36]:
```suggestion
OpenSearch Dashboards requires the Node.js runtime binary to run. One is included in the distribution packages available from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}.
```

File: _install-and-configure/install-dashboards/index.md [hunk 3]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
...
+OpenSearch Dashboards 2.8.0 and newer can use Node.js versions 14, 16, and 18. The distribution packages for OpenSearch Dashboards 2.10.0 and newer include Node.js 18 and 14 (for backward compatibility).
```
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md:37]: Insert a line space between lines 36 and 37.

File: _install-and-configure/install-dashboards/index.md [hunk 4]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
...
+   On Windows, if Node.js is installed to `C:\Program Files\nodejs` and the runtime binary is `C:\Program Files\nodejs\node.exe`:
+   ```powershell
+   set "NODE_HOME=C:\Program Files\nodejs"
+   # or using PowerShell:
+   $Env:NODE_HOME = 'C:\Program Files\nodejs'
+   ```
+
+   Please consult your operating system's documentation to make a persistent change to the environment variables.
+
+`bin/opensearch-dashboards`, the OpenSearch Dashboards start script will look for the Node.js runtime binary using `NODE_OSD_HOME`,
```
Comment (kolchfa-aws) [inline _install-and-configure/install-dashboards/index.md:65]:
```suggestion
The OpenSearch Dashboards start script,`bin/opensearch-dashboards`, searches for the Node.js runtime binary using `NODE_OSD_HOME`,
```

**Review by kolchfa-aws — APPROVED** [review]
Thanks, @AMoo-Miki!

File: _install-and-configure/install-dashboards/index.md [hunk 5]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
 
 Other Chromium-based browsers might work, as well. Internet Explorer and Microsoft Edge Legacy are **not** supported.
 
+## Node.js compatibility
+
+OpenSearch Dashboards requires the Node.js runtime binary to run and one is included in the distribution packages available from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}.
+OpenSearch Dashboards 2.8.0 and newer can use Node.js versions 14, 16, and 18. The distribution packages for OpenSearch Dashboards 2.10.0 and newer include Node.js 18 and 14 (for backward compatibility). 
+
+To use a Node.js runtime binary other that the ones included in the distribution packages, follow these steps:
```
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md:39]:
```suggestion
To use a Node.js runtime binary other than the ones included in the distribution packages, follow these steps:
```

File: _install-and-configure/install-dashboards/index.md [hunk 6]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
...
+
+1. Download and install [Node.js](https://nodejs.org/en/download){:target='\_blank'}; the compatible versions are `>=14.20.1 <19`.
+2. Set the installation path to the `NODE_HOME` or `NODE_OSD_HOME` environment variables.
+   
+   On UNIX, if Node.js is installed to `/usr/local/nodejs` and the runtime binary is `/usr/local/nodejs/bin/node`:
+    ```bash
+    export NODE_HOME=/usr/local/nodejs
+   ```
+
+   If Node.js installed using NVM and the runtime binary is `/Users/user/.nvm/versions/node/v18.19.0/bin/node`:
```
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md:49]:
```suggestion
    - If Node.js is installed using NVM and the runtime binary is `/Users/user/.nvm/versions/node/v18.19.0/bin/node`:
```

File: _install-and-configure/install-dashboards/index.md [hunk 7]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
...
+
+OpenSearch Dashboards requires the Node.js runtime binary to run and one is included in the distribution packages available from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}.
+OpenSearch Dashboards 2.8.0 and newer can use Node.js versions 14, 16, and 18. The distribution packages for OpenSearch Dashboards 2.10.0 and newer include Node.js 18 and 14 (for backward compatibility). 
+
+To use a Node.js runtime binary other that the ones included in the distribution packages, follow these steps:
+
+1. Download and install [Node.js](https://nodejs.org/en/download){:target='\_blank'}; the compatible versions are `>=14.20.1 <19`.
+2. Set the installation path to the `NODE_HOME` or `NODE_OSD_HOME` environment variables.
+   
+   On UNIX, if Node.js is installed to `/usr/local/nodejs` and the runtime binary is `/usr/local/nodejs/bin/node`:
```
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md:44]:
```suggestion
    - On UNIX, if Node.js is installed to `/usr/local/nodejs` and the runtime binary is `/usr/local/nodejs/bin/node`:
```

File: _install-and-configure/install-dashboards/index.md [hunk 8]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
...
+   ```
+
+   If Node.js installed using NVM and the runtime binary is `/Users/user/.nvm/versions/node/v18.19.0/bin/node`:
+   ```bash
+   export NODE_HOME=/Users/user/.nvm/versions/node/v18.19.0
+   # or, if NODE_HOME is used for something else:
+   export NODE_OSD_HOME=/Users/user/.nvm/versions/node/v18.19.0
+   ```
+
+   On Windows, if Node.js is installed to `C:\Program Files\nodejs` and the runtime binary is `C:\Program Files\nodejs\node.exe`:
```
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md:56]:
```suggestion
    - On Windows, if Node.js is installed to `C:\Program Files\nodejs` and the runtime binary is `C:\Program Files\nodejs\node.exe`:
```

File: _install-and-configure/install-dashboards/index.md (hunk 1 above)
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md]:
```suggestion
View your operating system's documentation to make a persistent change to the environment variables.
{: .note}
```

File: _install-and-configure/install-dashboards/index.md (hunk 4 above)
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md]:
```suggestion
`bin/opensearch-dashboards`, the OpenSearch Dashboards start script, looks for the Node.js runtime binary using `NODE_OSD_HOME`,
```

Comment (vagimeli) [reply _install-and-configure/install-dashboards/index.md]: Combine lines 65 and 66, as they are one sentence.

File: _install-and-configure/install-dashboards/index.md [hunk 9]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
...
+   ```powershell
+   set "NODE_HOME=C:\Program Files\nodejs"
+   # or using PowerShell:
+   $Env:NODE_HOME = 'C:\Program Files\nodejs'
+   ```
+
+   Please consult your operating system's documentation to make a persistent change to the environment variables.
+
+`bin/opensearch-dashboards`, the OpenSearch Dashboards start script will look for the Node.js runtime binary using `NODE_OSD_HOME`,
+and later `NODE_HOME`, before using the binaries included with the distribution packages. If a usable Node.js runtime binary is not found, the start script will
```
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md:66]:
```suggestion
and then `NODE_HOME`, before using the binaries included with the distribution packages. If a usable Node.js runtime binary is not found, the start script will
```

File: _install-and-configure/install-dashboards/index.md [hunk 10]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
...
+attempt to find one in the system-wide `PATH` before failing.
```
Comment (vagimeli) [inline _install-and-configure/install-dashboards/index.md:67]: Combine lines 65-67 into one line. There are breaks between words in sentences.

**Review by vagimeli — COMMENTED** [review]
Editorial review completed. Edits need to be made before merging.

File: _install-and-configure/install-dashboards/index.md [hunk 11]
```diff
@@ -31,6 +31,41 @@ OpenSearch Dashboards supports the following web browsers:
 
 Other Chromium-based browsers might work, as well. Internet Explorer and Microsoft Edge Legacy are **not** supported.
 
+## Node.js compatibility
+
+OpenSearch Dashboards requires the Node.js runtime binary to run. One is included in the distribution packages available from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}.
```
Comment (kolchfa-aws) [inline _install-and-configure/install-dashboards/index.md:36]:
```suggestion
OpenSearch Dashboards requires the Node.js runtime binary to run. One is included in the distribution packages available from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}.

```

File: _install-and-configure/install-dashboards/index.md [hunk 12]
```diff
@@ -31,6 +31,42 @@ OpenSearch Dashboards supports the following web browsers:
...
+   ```powershell
+   set "NODE_HOME=C:\Program Files\nodejs"
+   # or using PowerShell:
+   $Env:NODE_HOME = 'C:\Program Files\nodejs'
+   ```
+
+   Consult your operating system's documentation to make a persistent change to the environment variables.
+
+The OpenSearch Dashboards start script,`bin/opensearch-dashboards`, searches for the Node.js runtime binary using `NODE_OSD_HOME`,
+and then `NODE_HOME`, before using the binaries included with the distribution packages. If a usable Node.js runtime binary is not found, the start script will
```
Comment (kolchfa-aws) [inline _install-and-configure/install-dashboards/index.md:67]:
```suggestion
and then `NODE_HOME`, before using the binaries included with the distribution packages. If a usable Node.js runtime binary is not found, the start script will attempt to find one in the system-wide `PATH` before failing.
```

File: _install-and-configure/install-dashboards/index.md [hunk 13]
```diff
@@ -31,6 +31,42 @@ OpenSearch Dashboards supports the following web browsers:
...
+   set "NODE_HOME=C:\Program Files\nodejs"
+   # or using PowerShell:
+   $Env:NODE_HOME = 'C:\Program Files\nodejs'
+   ```
+
+   Consult your operating system's documentation to make a persistent change to the environment variables.
+
+The OpenSearch Dashboards start script,`bin/opensearch-dashboards`, searches for the Node.js runtime binary using `NODE_OSD_HOME`,
+and then `NODE_HOME`, before using the binaries included with the distribution packages. If a usable Node.js runtime binary is not found, the start script will attempt to find one in the system-wide `PATH` before failing.
+attempt to find one in the system-wide `PATH` before failing.
```
Comment (kolchfa-aws) [inline _install-and-configure/install-dashboards/index.md:68]:
```suggestion
```

**Pull request merged**